etc/schema.json

```diff
@@ -5507,27 +5507,39 @@
                             "type": "object",
                             "properties": {
                                 "flows_checked": {
+                                    "description":
+                                            "number of flows checked for timeout in the last pass",
                                     "type": "integer"
                                 },
                                 "flows_evicted": {
+                                    "description": "number of flows that were evicted or timed out",
                                     "type": "integer"
                                 },
                                 "flows_evicted_needs_work": {
+                                    "description":
+                                            "number of TCP flows that were handed to the workers",
```

Comment:
It's reassembled and returned to the thread it belongs to. I'm unsure if those are worker threads though or it could be better explained in a one liner..

Reply:
no reassembly happens, it's just checked if reassembly/detect/logging still needs to complete, and if so the flow is handed to it's worker. The worker will do the actual work.

Reply:
"Reassembled" seems to be used in these fn names and in the context of flow I thought reassembling flow meant sending them to their respective threads. Would that be a correct assumption?
Thank you. Your explanation makes sense to me.

Reply:
Would be good to rename the functions to a more generic "Work To Do" terminology.

Reply:
ok. thank you.